spec/requests/todos_spec.rb

```diff
@@ -0,0 +1,100 @@
+require 'rails_helper'
+
+RSpec.describe "Todos API", type: :request do
+	let!(:todos) {create_list(:todo, 10)}
+	let(:todo_id) {todos.first.id}
+
+	describe 'GET /todos index method' do
+		before { get '/todos'}
+
+		it 'returns todos' do
+			expect(json).not_to be_empty
+			expect(json.size).to eq(10)
+		end
+
+		it 'returns status code' do
+			expect(response).to have_http_status(200)
```

Comment:
We also need to test that all todos are returning from api response. Something like this
`expect(JSON.parse(response.body)).to eq(-----)`